kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -268,24 +268,20 @@
     position: absolute;
     bottom: 0;
     width: 100%;
-    padding: $margin;
+    padding: $margin 24px;
```

Comment:
Is there a reason to keep `$margin` around? It doesn't appear to be used except here.

We can delete it and just do:


```suggestion
    padding: 16px 24px;
```

Reply:
now that `$margin` is completely unused, the definitions should be deleted